packages/create-svelte/bin.js

```diff
@@ -92,6 +92,14 @@ async function main() {
 					initial: false,
 					active: 'Yes',
 					inactive: 'No'
+				},
+				{
+					type: 'toggle',
+					name: 'playwright',
+					message: 'Add Playwright for testing?',
```

Comment:
Maybe we should expand this message in case we offer Vitest or similar in the future alongside Playwright?
```suggestion
					message: 'Add Playwright for end-to-end browser testing?',
```